src/EditorFeatures/Core/Implementation/TodoComment/TodoCommentTokens.cs

```diff
@@ -73,10 +73,9 @@ public TokenInfo(string optionText, ImmutableArray<TodoCommentDescriptor> tokens
 
         private TokenInfo _lastTokenInfo;
 
-        public async Task<ImmutableArray<TodoCommentDescriptor>> GetTokensAsync(Document document, CancellationToken cancellationToken)
+        public ImmutableArray<TodoCommentDescriptor> GetTokens(Document document, CancellationToken cancellationToken)
         {
-            var documentOptions = await document.GetOptionsAsync(cancellationToken).ConfigureAwait(false);
-            var optionText = documentOptions.GetOption(TodoCommentOptions.TokenList);
+            var optionText = document.Project.Solution.Options.GetOption(TodoCommentOptions.TokenList);
```

Comment:
For now, yes. Once we figure out a way to represent that (and we've probably moved that provider out of proc for performance), we can put this back.